src/Symfony/Component/HttpKernel/Event/ControllerArgumentsEvent.php

```diff
@@ -28,8 +28,8 @@
  */
 final class ControllerArgumentsEvent extends KernelEvent
 {
-    private $controller;
-    private $arguments;
+    private string|array|object $controller;
```

Comment:
I don't see the value in this workaround for missing callable type

Reply:
Yeah, I know. It's more a reminder to ourselves that we haven't forgotten to add a type here, I guess. I can remove it.